docs/sources/reference/templating.md

```diff
@@ -165,6 +165,23 @@ Option | Description
 *Regex* | Regex to filter or capture specific parts of the names return by your data source query. Optional.
 *Sort* | Define sort order for options in dropdown. **Disabled** means that the order of options returned by your data source query will be used.
 
+#### Using `$__searchFilter` to filter query results
+> Only supported for Graphite, MySQL and Postgres data sources
+
+Using `$__searchFilter` in the query field will filter the query result based on what the user types in the dropdown select box
+
+Examples of how to use `$__searchFilter` as part of the query
+
+Graphite:
+```bash
+server.$__searchFilter
+```
+
+MySQL:
+```bash
+SELECT server FROM servers WHERE server LIKE $__searchFilter
+```
+
```

Comment:
Not sure if this belongs to the general page or if I should move this to the specific data sources.

Reply:
Tricky, since it's only going to be supported by some data sources maybe it makes sense to have it in each data source doc.

Reply:
Good to mention that the default empty __searchFilter state will be a wildcard query

Reply:
Agree, I've moved documentation to data source specific pages instead